annotated-json.js

```diff
@@ -181,10 +225,27 @@ function _render (annotations, json) {
     }
   }
 
+  // fill in any keys the user may have added directly
+  keys = Object.keys(json)
+  for (i = 0; i < keys.length; ++i) {
+    name = keys[i]
+    if (!(name in foundKeys)) {
+      obj = {}
+      obj[name] = json[name]
+      out.push(obj)
+    }
+  }
```

Comment:
... not sure this is exactly what we want. If the user adds an object it'll just be added directly... i.e. not translated into array-form.